.github/workflows/pr.yaml

```
@@ -953,7 +976,34 @@ jobs:
+      id-token: write
+      packages: read
+      pull-requests: read
+    secrets: inherit # zizmor: ignore[secrets-inherit]
+    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@main
+    # Run whenever the shards ran (pass or fail) so we always get a diff, but
+    # not when pandas-tests was skipped. continue-on-error keeps this purely
+    # informational job from ever blocking the PR.
+    if: ${{ !cancelled() && needs.pandas-tests.result != 'skipped' }}
+    with:
+      build_type: pull-request
+      branch: ${{ inputs.branch }}
```

actionlint (1.7.12)

[error] property "branch" is not defined in object type {}

(expression)